projects/Mallard/src/helpers/push-notifications.ts

```diff
@@ -126,18 +126,19 @@ const pushNotifcationRegistration = () => {
                     pushTracking('pushDownloadComplete', 'completed')
 
                     notificationTracking(notificationId, 'downloaded')
+
+                    // No matter what happens, always clear up old issues
+                    await clearOldIssues()
                     // required on iOS only (see fetchCompletionHandler docs: https://facebook.github.io/react-native/docs/pushnotificationios.html)
                     notification.finish(PushNotificationIOS.FetchResult.NoData)
                 } catch (e) {
                     console.log(
                         `Push notification unable to download: ${e.message}`,
                     )
                     errorService.captureException(e)
+                    await clearOldIssues()
```

Comment:
Potentially we can put this and `notification.finish(PushNotificationIOS.FetchResult.NoData)` in a `finally` so that if we need to edit the things that always run, there's an obvious place for me.